py/plugins/google-genai/src/genkit/plugins/google_genai/google.py

```diff
@@ -322,20 +323,23 @@ def _resolve_model(self, ai: GenkitRegistry, name: str) -> None:
                 prefix indicating it's from a specific plugin (e.g., 'vertexai/gemini-pro').
         """
         _clean_name = name.replace(VERTEXAI_PLUGIN_NAME + '/', '') if name.startswith(VERTEXAI_PLUGIN_NAME) else name
-        model_ref = google_model_info(_clean_name)
 
         if 'image' in _clean_name.lower():
```

Comment:
in JS we differentiate on `imagen` prefix: https://github.com/firebase/genkit/blob/7cfec0a7fdf16f2c3d7f3e68212a96de4ce23c8e/js/plugins/vertexai/src/index.ts#L185
just to be consistent.